modules/doi/includes/utilities.inc

```diff
@@ -176,6 +176,7 @@ function islandora_doi_perform_request($id_or_url) {
  */
 function islandora_doi_crossref_translator($crossref_data) {
   // Parse the source into a DOMDocument.
+  watchdog('test', htmlentities($crossref_data));
```

Comment:
I guess the `watchdog` call should not be here, right?

Reply:
Correct! Removing now.